feature-detects/emoji.js

```diff
@@ -10,12 +10,14 @@ Detects support for emoji character sets.
 define(['Modernizr', 'createElement', 'test/canvastext'], function( Modernizr, createElement ) {
   Modernizr.addTest('emoji', function() {
     if (!Modernizr.canvastext) return false;
+    var pixelRatio = window.devicePixelRatio || 1;
+    var offset = 8 * pixelRatio;
     var node = createElement('canvas'),
     ctx = node.getContext('2d');
```

Comment:
I think it's just a style thing? `ctx` is okay because there's a comma on the previous line, it should have been indented. You might want to add JSCS to the build process for these issues now that `onevar` is removed from JSHint.